features/admin.flow-builder-core.v1/components/flow-builder-page-skeleton/flow-builder-page-header.tsx

```diff
@@ -133,6 +134,34 @@ const FlowBuilderPageHeader: FunctionComponent<FlowBuilderPageHeaderProps> = ({
         };
     }, []);
 
+    /**
+     * Resolves the tooltip content for the toggle switch based on the flow type.
+     */
+    const toggleTooltipContent: { enable: string; disable: string } = useMemo(() => {
```

Comment:
Shall we just keep a generic content for the tooltip. Without mentioning the flow type.
Coz we need to remove the `@wso2is/admin.flows.v1` dependency in `flow-builder-core.v1` to avoid cercular dependencies.